Week3/week3HomeWork/Exercise3/fortuneTeller.js

```diff
@@ -0,0 +1,39 @@
+const numChildren = [1, 2, 3, 4, 5];
+const partnerNames = ["Jessica", "Wilma", "Nadia", "Susan", "Angelina"];
+const locations = ["Syria", "London", "Turkey", "Netherland", "Franca"];
+const jobs = ["accountant", "brogrammer", "salesman", "businessman", "manager"];
+
+function tellFortune(child, partener, location, job) {
+    const numChildren = [1, 2, 3, 4, 5];
+    const partnerNames = ["Jessica", "Wilma", "Nadia", "Susan", "Angelina"];
+    const locations = ["Syria", "London", "Turkey", "Netherland", "Franca"];
+    const jobs = [
+        "accountant",
+        "brogrammer",
+        "salesman",
+        "businessman",
+        "manager"
+    ];
+    child = numChildren[Math.floor(Math.random() * 5)];
+    partener = partnerNames[Math.floor(Math.random() * 5)];
+    location = locations[Math.floor(Math.random() * 5)];
+    job = jobs[Math.floor(Math.random() * 5)];
+    console.log(
+        `You will be a ${job} in ${location}, and married to ${partener} with ${child} Kids`
+    );
+}
+
+tellFortune();
+
+// Another way for the solution
+function newTellFortune(child, partener, location, job) {
```

Comment:
I don't think you have to add this one:) 

But with this code, what if I do the following:

```
newTellFortune(52, 52, 32, 42);
```

? Does it then still work?

Reply:
You will be a **undefined** in undefined, and married to **undefined** with undefined Kids 
it give undefined